src/actions/files.js

```diff
@@ -37,6 +37,19 @@ export function getFilesForPost(postId) {
     };
 }
 
+export function getMissingFilesForPost(postId) {
```

Comment:
Only want to get them if we don't have them, it'll be a perf problem otherwise since this is called on mount